lib/nanoc-redirector.rb

```diff
@@ -17,10 +17,8 @@ def self.process(item, dest)
         redirects.each do |redirect|
           content = NanocRedirector.redirect_template(dest)
           dir = "output/#{redirect}"
-          unless File.directory?(dir)
-            FileUtils.mkdir_p(dir)
-            File.write("#{dir}/index.html", content)
-          end
+          FileUtils.mkdir_p(dir) unless File.directory?(dir)
+          File.write("#{dir}/index.html", content) unless File.exist? "#{dir}/index.html"
```

Comment:
Even though I totally didn't do it, could we fix these to become `File.join` in case one day we have a Windows colleague?